### DISCUSSION TITLE
It is not possible to run all samples

### DISCUSSION THREAD

**Discussion by Vjze** [Relm4/Relm4#652] [General]
thread 'main' panicked at relm4/src/lib.rs:117:17:
called `Result::unwrap()` on an `Err` value: BoolError { message: "Failed to initialize GTK", filename: "/Users/wjz/.cargo/registry/src/index.crates.io-6f17d22bba15001f/gtk4-0.8.2/src/rt.rs", function: "gtk4::rt::init", line: 158 }

**Comment by AaronErhardt** [comment]
Can you provide more context? Which command did you run and have you confirmed that GTK is installed properly on your system?

**Reply by Vjze** [reply]
cargo run --examples simple_manual,  I'm using MacOS 14 and installed GTK4 via brew, and all dependencies are compiled correctly when I run the command, and I get an error when I run it at the end.

**Reply by AaronErhardt** [reply]
Apparently, GTK was not installed properly or is somehow not working as expected. The error originates from gtk-rs when [`gtk_init_check` is called via FFI](https://github.com/gtk-rs/gtk4-rs/blob/0.8.2/gtk4/src/rt.rs#L158). This method is provided by the GTK library and is not influenced by any surrounding Rust code. Therefore, opening an issue at the GTK repository would make more sense.
